src/js/components/stock-movement-wizard/modals/EditPickModal.jsx

```diff
@@ -26,11 +26,7 @@ import { formatDate } from 'utils/translation-utils';
 const FIELDS = {
   reasonCode: {
     type: SelectField,
-    label: () => (
-      <label htmlFor="reasonCode" className="ml-3 text-center">
-        <Translate id="react.stockMovement.reasonCode.label" defaultMessage="Reason code" />
-      </label>
-    ),
+    label: 'react.stockMovement.reasonCode.label',
     defaultMessage: 'Reason code',
     attributes: {
       className: 'mb-2',
```

Comment:
do I need to move the "ml-3 text-center" className values here?? I'm a frontend baby :baby:

Reply:
if you moved it, it would be applied to the SelectField (the wrapper around the label element), not to the label itself, so let's skip it, as it's only a small margin that is not even noticeable.